code/game/gamemodes/revolution/revolution.dm

```diff
@@ -374,7 +201,7 @@
 		text += "<br>"
 		to_chat(world, text)
 
-/datum/game_mode/revolution/set_scoreboard_vars()
+/datum/game_mode/revolution/set_scoreboard_vars() // is this ever even called?
```

Comment:
is it?

Reply:
I... have no clue. But I am scared of changing it, some other gamemodes use it, but I'm not sure if its ever actually called still

Reply:
I'm going to leave it as is then, removal of it wholly would be out of scope